pywbemtools/pywbemcli/_cmd_namespace.py

```diff
@@ -0,0 +1,210 @@
+# (C) Copyright 2020 IBM Corp.
+# (C) Copyright 2020 Inova Development Inc.
+# All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Click Command definition for the 'namespace' command group which includes
+commands for create, delete, list, etc. of the CIM namespaces on a WBEM server.
+
+NOTE: Commands are ordered in help display by their order in this file.
+"""
+
+from __future__ import absolute_import, print_function
+
+import click
+
+from pywbem import Error
+from .pywbemcli import cli
+from ._common import CMD_OPTS_TXT, GENERAL_OPTS_TXT, SUBCMD_HELP_TXT, \
+    DEFAULT_TABLE_FORMAT, output_format_is_table, format_table, display_text, \
+    raise_pywbem_error_exception, validate_output_format
+from ._common_options import add_options, help_option
+from ._click_extensions import PywbemcliGroup, PywbemcliCommand
+
+
+##########################################################################
+#
+#   Click command group and command definitions
+#   These decorated functions implement the commands, arguments, and
+#   options for the top-level 'namespace' command group
+#
+###########################################################################
+
+@cli.group('namespace', cls=PywbemcliGroup, options_metavar=GENERAL_OPTS_TXT,
+           subcommand_metavar=SUBCMD_HELP_TXT)
+@add_options(help_option)
+def namespace_group():
+    """
+    Command group for CIM namespaces.
+
+    This command group defines commands to create, delete and list namespaces
+    in a WBEM server.
+
+    In addition to the command-specific options shown in this help text, the
+    general options (see 'pywbemcli --help') can also be specified before the
+    'class' keyword.
```

Comment:
The word 'class' should be 'namespace' since this is the namespace group

Reply:
DONE